iopipe/mock_system.py

```diff
@@ -1,6 +1,6 @@
 import uuid
 
-from .system import read_arch, read_hostname  # noqa
+from .system import read_hostname  # noqa
```

Comment:
do you need #noqa now that you removed an unused import?

Reply:
Yeah, the `read_hostname` is still not used in the module.

Reply:
@kolanos then why is it imported?

Reply:
Because it is used by a module that imports this module.

Reply:
Ahh got it now. Thank you!